sdks/unreal/Agones/Source/Agones/Private/AgonesComponent.cpp

```diff
@@ -146,6 +192,45 @@ void UAgonesComponent::GameServer(const FGameServerDelegate SuccessDelegate, con
 	Request->ProcessRequest();
 }
 
+bool UAgonesComponent::DeserializeAndBroadcastWatch(FString const& JsonString)
+{
+    TSharedPtr<FJsonObject> JsonObject;
+    const TSharedPtr<FJsonObject>* ResultObject = nullptr;
+    TSharedRef<TJsonReader<TCHAR>> const JsonReader = TJsonReaderFactory<TCHAR>::Create(JsonString);
+    if (FJsonSerializer::Deserialize(JsonReader, JsonObject) && JsonObject.IsValid() &&
+        JsonObject->TryGetObjectField(TEXT("result"), ResultObject) && ResultObject->IsValid())

```

Comment:
nit  - is more readable if we split these over lines but clangformat might complain :/